### PR TITLE
Add support for coinbase wallet

### PR DESCRIPTION
**Pull request by mveytsman**
Tested it locally.

Screen Shot 2022-02-20 at 2 37 56 PM

I think this looks a little weird with the missing corner, and there's no option to change the CSS I can find easily. Should we add a fourth option? heres the list: https://github.com/Web3Modal/web3modal#provider-options

### CODE REVIEW

File: components/ConnectWallet.vue [hunk 1]
```diff
@@ -35,14 +35,23 @@ export default {
...
-              infuraId: this.networkConfig.web3Fallback.split('/').pop()
-            }
-          }
+              infuraId: infuraId,
+            },
+          },
+          walletlink: {
+            package: WalletLink,
+            options: {
+              appName: "Tousand Ether Homepage", // Required
```
Comment (shazow) [inline components/ConnectWallet.vue:50]:
```suggestion
              appName: "Thousand Ether Homepage", // Required
```

**Review by shazow — CHANGES_REQUESTED** [review]
Hm doesn't work for me on staging, does it work for you? The Wallet Connect isn't popping up at all with MetaMask. Prod still works for me.

https://deploy-preview-124--thousandetherhomepage.netlify.app/

**Comment by mveytsman** [comment]
Dang, my editor ran the code through eslint, and in trying to not check in all the formatting changes I missed an import. It's working for me now in staging. You?

**Review by shazow — APPROVED** [review]
Works now, thanks!

Also don't mind eslint, lint on!

**Pull request merged**